style.css

```diff
@@ -59,4 +68,204 @@
   text-align: center;
   margin-top: 50px;
   margin-bottom: 60px;
-}
\ No newline at end of file
+}
+
+.works {
+  margin-top: 0;
+  text-align: center;
+}
+
+.title {
+  background-image: url("./images/Indicator.svg");
+  background-repeat: no-repeat;
+  background-position: bottom;
+  width: 327px;
+  height: 125px;
+  margin: 0 auto;
+  margin-top: 0;
+  margin-bottom: 62px;
+}
+
+.work {
+  display: grid;
+  align-content: end;
+  width: 327px;
+  height: 474px;
+  border: 1px solid #d0d9d4;
+  background-color: #ebf0ee;
+  border-radius: 10px;
+  margin: 0 auto;
+  margin-bottom: 24px;
+}
```

Comment:
> Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

- [ ] Your work card looks like the given design but there are changes you need to improve on here.
- [ ] According to the requirements, you are meant to use Flexbox and not Grid to lay out the content of individual work cards, I'll suggest you use Flexbox as required :+1: :smile: